lib/install/management/store_files.go

```diff
@@ -189,3 +191,44 @@ func (d *Dispatcher) createVolumeStores(conf *metadata.VirtualContainerHostConfi
 	}
 	return nil
 }
+
+func (d *Dispatcher) deleteVolumeStoreIfForced(conf *metadata.VirtualContainerHostConfigSpec) {
+	if d.force {
+		for label, url := range conf.VolumeLocations {
+			// separate the host (datastore name) from the path in the provided path URL
+			pathComponents := strings.SplitN(url.Path, " ", 2)
```

Comment:
This format surprises me given our discussion earlier today - is this PR based off another that changes this or am I mis-remembering the current format?